src/utils/stark/stark-curve.ts

```diff
@@ -60,17 +65,28 @@ function hashKeyWithIndex(key: string, index: number): BN {
   );
 }
 
-function grindKey(key: BN, keyValLimit: BN) {
+/*
+ This function receives a key seed and produces an appropriate StarkEx key from a uniform
+ distribution.
+ Although it is possible to define a StarkEx key as a residue between the StarkEx EC order and a
```

Comment:
An equation here might help make this more intuitive `Stark private key = Random 256-bit field element (mod Stark EC order)`.